### PR TITLE
Project 12 - Rebecca Philipson & Christoffer Torninger

### PR DESCRIPTION
**Pull request by Rephili**
Link to my deployed project:
https://movies-site-rephili-vrill.netlify.app/

### CODE REVIEW

File: code/src/App.js [hunk 1]
```diff
@@ -1,9 +1,39 @@
...
-    <div>
-      Find me in src/app.js!
-    </div>
-  )
-}
+    <>
+    {loading && <Loading />}
+    <BrowserRouter>
+      <Switch>
+        <Route exact path="/" render={() => <MovieList movies={movies} />} />
```
Comment (Jsfrulle) [inline code/src/App.js:29]: Nice how you guys useing the route here :D

File: code/src/App.js [hunk 2]
```diff
@@ -1,9 +1,39 @@
...
+import Loading from "./components/Loading";
+import { MovieList } from "./components/MovieList";
+import { MovieDetails } from "./components/MovieDetails";
+import NotFound from "./components/NotFound";
 
 export const App = () => {
+  const [movies, setMovies] = useState([]);
+  const [loading, setLoading] = useState(true)
+
+  useEffect(() => {
```
Comment (Jsfrulle) [inline code/src/App.js:15]: If you want the loading component to showing just add this :) its not needed if you dont want that for this project

.then((res) => res.json())
      .then((json) => {
        setTimeout(() => setLoading(false), 2000);
        setMovies(json.results);
      });

File: code/src/utils/urls.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+// Api key and urls to fetch the movie list and the details
+
+export const API_KEY = "a42559943824c6a02203973f57136d18";
+export const MOVIES_URL = (API_KEY) =>
+  `https://api.themoviedb.org/3/movie/popular?api_key=${API_KEY}&language=en-US&page=1`;
+export const MOVIE_DETAILS_URL = (API_KEY, movieId) =>
+  `https://api.themoviedb.org/3/movie/${movieId}?api_key=${API_KEY}&language=en-US`;
```
Comment (Jsfrulle) [inline code/src/utils/urls.js:7]: A really nice project whit a clean design! everything is working and the code is easy to read! Goood Work!!!!
- [YES ]  Am I able to understand the code easily?
- [NO ]  Does the code have lots of duplicates? Could it be tidied up?
- [ YES it works]  Does it work or does anything look broken?
- [ YES]  Does it follow the general and blue requirements?

File: code/src/components/MovieDetails.css [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+  padding-left: 20px;
+}
+
+.vote-average {
+  margin-left: 10px;
+  color: red;
+  font-size: 18px;
+}
+
+@media (max-width: 767px) {
```
[the remark this answers is not in the document]
Comment (Rephili) [reply code/src/components/MovieDetails.css:79]: Thanks for the tip! :)

File: code/src/App.js (hunk 2 above)
Comment (Rephili) [reply code/src/App.js:15]: Thank you Jessica! Tried adding that now, but it still doesn't show so probably there is something more missing. Will have to look into that. :)

File: code/src/utils/urls.js (hunk 3 above)
Comment (Rephili) [reply code/src/utils/urls.js:7]: Tack Jessica for your feedback :)

**Pull request closed without merging**